pkg/maps/ctmap/gc/gc.go

```diff
@@ -239,9 +242,34 @@ func createGCFilter(initialScan bool, restoredEndpoints []*endpoint.Endpoint, em
 	if initialScan {
 		filter.ValidIPs = map[string]struct{}{}
 		for _, ep := range restoredEndpoints {
+			if ep.IsHost() {
+				continue
+			}
 			filter.ValidIPs[ep.IPv6.String()] = struct{}{}
 			filter.ValidIPs[ep.IPv4.String()] = struct{}{}
 		}
+
+		// Once the host firewall is enabled, we will start tracking (and
+		// potentially enforcing policies) on all connections to and from the
+		// host IP addresses. Thus, we also need to avoid GCing the host IPs.
+		if option.Config.EnableHostFirewall {
+			addrs, err := nodeAddressing.IPv4().LocalAddresses()
+			if err != nil {
+				log.WithError(err).Warning("Unable to list local IPv4 addresses")
+			}
+			addrsV6, err := nodeAddressing.IPv4().LocalAddresses()
+			if err != nil {
+				log.WithError(err).Warning("Unable to list local IPv4 addresses")
```

Comment:
`nodeAddressing.IPv4().LocalAddresses()` => `nodeAddressing.IPv6().LocalAddresses()`?
`Unable to list local IPv4 addresses` => `Unable to list local IPv6 addresses`?

Reply:
:facepalm: 

Thanks, fixed!